core/request.go

```diff
@@ -130,6 +130,23 @@ func (r *RequestAccessor) ProxyEventToHTTPRequest(req events.APIGatewayProxyRequ
 			queryCnt++
 		}
 	}
+	if len(req.MultiValueQueryStringParameters) > 0 {
```

Comment:
This will add the same parameter twice. If the query string parameter is passed in, API Gateway will populate both the old `queryString` map and the `multiValueQueryString`. I think we should just drop support for the `queryString` map and only support the multi-value one. ALB produces the same multivalue map - although you have to enable it explicitly.

Reply:
Sure.